### ISSUE TITLE
tbl_JobYield missing error

### ISSUE THREAD

**Issue by porsouw** [microsoft/Code-Search#84]
Truncate table search.tbl_JobYield fails in CleanUpCollectionIndexingState_IndexDelete.sql. 
Seems that the table is (no longer) present?

**Comment by bittusarkar** [comment]
Yes, this script needs to be fixed. May I ask how did you land up using this script?
